ui/desktop/app/styles/app.scss

```diff
@@ -502,7 +502,18 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
-.session-details-header,
+.session-details-header {
```

Comment:
Can we consolidate these styes to a more generic class name and just apply the class multiple times rather than using `&, h1` since they're exactly the same styles